frontend/src/metabase-lib/metadata.ts

```diff
@@ -47,3 +47,26 @@ export function groupColumns(columns: ColumnMetadata[]): ColumnGroup {
 export function getColumnsGroupColumns(group: ColumnGroup): ColumnMetadata[] {
   return ML_ColumnGroup.columns_group_columns(group);
 }
+
+export function describeTemporalUnit(
+  unit: string | null = null,
+  n: number = 1,
+): string {
+  return ML.describe_temporal_unit(n, unit);
+}
+
+type IntervalAmount = number | "current" | "next" | "last";
```

Comment:
FWIW, if there's a chance this type might be used externally or should be shared, it's worth moving it to `metabase-lib/types`. No worries if it's not the case tho